manim/mobject/three_d/three_dimensions.py

```diff
@@ -131,12 +140,11 @@ def __init__(
         )
         self.resolution = resolution
         self.surface_piece_config = surface_piece_config
-        if checkerboard_colors:
-            self.checkerboard_colors: list[ManimColor] = [
-                ManimColor(x) for x in checkerboard_colors
-            ]
-        else:
+        self.checkerboard_colors: list[ManimColor] | Literal[False]
+        if checkerboard_colors is False:
             self.checkerboard_colors = checkerboard_colors
+        else:
+            self.checkerboard_colors = [ManimColor(i) for i in checkerboard_colors]
```

Comment:
Following the previous suggestion, this could be something like:
```suggestion
        self.checkerboard_colors: list[ManimColor] | None
        if checkerboard_colors is None:
            self.checkerboard_colors = checkerboard_colors
        else:
            self.checkerboard_colors = [ManimColor(i) for i in checkerboard_colors]
```
or:

```suggestion
        self.checkerboard_colors = (
            [ManimColor(i) for i in checkerboard_colors]
            if checkerboard_colors is not None else None
        )
```